kubernetes/stream_cluster.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
   namespace: stream-clients-test
 spec:
   replicas: 1
-  image: rabbitmq:3.11.11-management
+  image: rabbitmq:3.11.17-management
```

Comment:
This is fine, although latest 3.11 is 3.11.20. Any reason to not bump to the latest? Or even to tag `rabbitmq:3.11-management` that tracks the latest patch?